app/src/util/tools.ts

```diff
@@ -226,6 +228,26 @@ export const isAddress = (address: string): boolean => {
   return true
 }
 
+export const waitForConfirmations = async (
+  transaction: any,
+  cpk: CPKService,
+  setNumberOfConfirmations: any,
```

Comment:
Shouldn't this be type number?

Reply:
which one? Well transaction is special object that contains transaction data...and setNumberOfConfirmations is hook from useState

Reply:
Ahhh, oh yeah I misread this